gatsby/src/components/ChapterBody/ChapterBody.module.scss

```diff
@@ -66,3 +66,11 @@
     display: none;
   }
 }
+
+@media print {
```

Comment:
I think we should also hide the down chevron and show link - what do you think @colinmazu?

Reply:
yep agree